lib/omnitrade_client.dart

```diff
@@ -0,0 +1,24 @@
+library omnitrade_client;
+
+import 'package:chopper/chopper.dart';
+import 'src/services.dart';
+import 'package:http/http.dart' as http;
+
+class OmniCredentials {}
+
+/// A OmniClient.
+class OmniTradeClient extends OmniServices {
+  OmniTradeClient({this.httpClient, this.credentials}) {
+    credentials ??= OmniCredentials();
```

Comment:
this variable is coming from someplace? if not you don't need the `??=`

Reply:
This variable comes from the named parameter of the constructor, so if the user doesn't provide it `??=` will create an instance for public requests